media/js/base/fxa-utm-referral.js

```diff
@@ -65,10 +65,13 @@ if (typeof window.Mozilla === 'undefined') {
          * the option of passing a `source` parameter to help connect attribution with FxA link referrals.
          */
         if (Object.prototype.hasOwnProperty.call(finalParams, 'source')) {
-            if (finalParams['source'].indexOf('whatsnew') !== -1 || finalParams['source'].indexOf('welcome') !== -1) {
+            if (finalParams['source'].indexOf('whatsnew') !== -1 || finalParams['source'].indexOf('welcome') !== -1 || finalParams['source'].indexOf('vpn-info') !== -1) {
 
                 // utm_source and entrypoint should always be consistent and omit a version number.
-                if (finalParams['source'].indexOf('whatsnew') !== -1 ) {
+                if (finalParams['source'].indexOf('vpn-info') !== -1 ) {
+                    finalParams['utm_source'] = 'www.mozilla.org-vpn-info';
+                    finalParams['entrypoint'] = 'www.mozilla.org-vpn-info';
+                } else if  (finalParams['source'].indexOf('whatsnew') !== -1 ) {
```

Comment:
@alexgibson when you get a second - let me know if this looks correct to you

Reply:
I think this looks fine for this PR 👍 

I'm working on a better solution for this in https://github.com/mozilla/bedrock/issues/10409 but for now this works! 💯